wire-ios-utilities/Source/Abstractions/UserNotificationCenter/UserNotificationCenterWrapper.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UserNotifications
+
+public struct UserNotificationCenterWrapper: UserNotificationCenterAbstraction {
+
+    private var userNotificationCenter: UNUserNotificationCenter
+
+    public var delegate: (any UNUserNotificationCenterDelegate)?
```

Comment:
A delegate should be `weak` to avoid retain cycles.

Reply:
Thanks, this wasn't supposed to be a stored property. Fixed.